gnome-initial-setup/pages/software/gis-software-page.c

```diff
@@ -88,12 +88,12 @@ gis_software_page_dispose (GObject *object)
   G_OBJECT_CLASS (gis_software_page_parent_class)->dispose (object);
 }
 
+#ifdef ENABLE_SOFTWARE_SOURCES
```

Comment:
Should this change go upstream?

Reply:
Yes, I opened https://gitlab.gnome.org/GNOME/gnome-initial-setup/merge_requests/19